src/main/java/io/jenkins/plugins/folderauth/FolderAuthorizationStrategyManagementLink.java

```diff
@@ -83,6 +83,19 @@ public String getDisplayName() {
         return Messages.FolderBasedAuthorizationStrategy_DisplayName();
     }
 
+    /**
+     * Name of the category for this management link. Exists so that plugins with core dependency pre-dating the version
+     * when this was introduced can define a category.
+     *
+     * TODO when the core version is >2.226 change this to override {@code getCategory()} instead
```

Comment:
```suggestion
     * TODO when the core version is &gt;2.226 change this to override {@code getCategory()} instead
```

This causes the Javadoc generation to fail

```
[2020-05-25T08:46:44.926Z] [ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.1.1:jar (attach-javadocs) on project folder-auth: MavenReportException: Error while generating Javadoc: 

[2020-05-25T08:46:44.926Z] [ERROR] Exit code: 1 - /home/jenkins/workspace/Plugins_folder-auth-plugin_PR-56/src/main/java/io/jenkins/plugins/folderauth/FolderAuthorizationStrategyManagementLink.java:90: error: bad use of '>'

[2020-05-25T08:46:44.926Z] [ERROR]      * TODO when the core version is >2.226 change this to override {@code getCategory()} instead
```